### DISCUSSION TITLE
이번주(3월 25일)은 참석이 어려울 것 같습니다.

### DISCUSSION THREAD

**Discussion by momoci99** [ThinkAboutSoftware/AcademicConference#186] [General]
안녕하세요 회원님들 이번주 모임은 개인 사정으로 인해 참석이 어려울 것 같습니다.😢 죄송합니다!

**Comment by jongfeel** [comment]
알겠습니다. 푹 쉬시고 다음 번에 뵈요.
그래도 책 읽고 논의 내용을 늦게라도 올려 주시면 리뷰 해 드리겠습니다.
